app/src/test/java/ingsw/group1/findmyphone/contacts/ContactConverterTest.java

```diff
@@ -0,0 +1,69 @@
+package ingsw.group1.findmyphone.contacts;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import ingsw.group1.msglibrary.SMSPeer;
+
+import static org.junit.Assert.*;
+
+/**
+ * Test for {@link ContactConverter}
+ *
+ * @author Giorgia Bortoletti
+ */
+public class ContactConverterTest {
+
+    private static final String EX_VALID_ADDRESS = "+393478989890"; //for contact and peer
+    private static final String CONTACT_VALID_NAME = "NewContact";
+    private static final String CONTACT_EMPTY_NAME = "";
+
+    private Contact contactWithoutName, contactWithName; //a contact may or may not have a name
+    private SMSPeer peerTest;
+
+    @Before
+    public void createPeerAndContact(){
+        contactWithoutName = new Contact(EX_VALID_ADDRESS, CONTACT_EMPTY_NAME);
+        contactWithName = new Contact(EX_VALID_ADDRESS, CONTACT_VALID_NAME);
+        peerTest = new SMSPeer(EX_VALID_ADDRESS);
+    }
+
+    /**
+     * Test {@link ContactConverter#contactFromSMSPeer(SMSPeer)}
+     *
+     * Conversion from a SMSPeer to a Contact without a name
+     */
+    @Test
+    public void contactFromSMSPeer_withEmptyName() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The JUnit 4 test method name 'contactFromSMSPeer_withEmptyName' doesn't match '[a-z][a-zA-Z0-9]*'](https://app.codacy.com/manual/RicDeZen/ingsw-group1-findmyphone/pullRequest?prid=4929954)